go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/opencontainers/image-spec
 
-go 1.18
+go 1.19
```

Comment:
Is there a compelling reason to bump this too?  (In doing so, we force anyone who wants to import this to bump as well, which doesn't seem particularly downstream friendly, especially since we're not actually using anything new from 1.19+.)

I'm not opposed to the other half of this change (although I'd personally keep 1.18 in there too until we have a real reason not to).

Reply:
I can pull it if this is blocking the merge. There's no specific requirement that I'm aware of.

My general assumption is anyone with an older compiler requirement will also be using an older version of the spec. E.g. if they are running a build of an old project, the go.mod/go.sum will pick the image-spec no more recent than when that old project received its last commit.

In the future, Go will transparently download and use a newer version of the compiler, and alter it's behavior based on the go.mod version of each package, making this less of an issue.

Reply:
I've reverted that line for now, we can revisit it later if needed/wanted.